examples/golang/add-object.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"bufio"
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+)
+
+const (
+	id  = "AWS_ACCESS_KEY_ID"
+	key = "AWS_SECRET_ACCESS_KEY"
+)
+
+func main() {
+	endPoint := flag.String("endpoint", "", "region of request")
```

Comment:
"endpoint" and "region" are not the same thing in Ceph.
in AWS when you select a region (e.g. us-east) you have a default endpoint that serves this region.
in the case of Ceph, the endpoint is the IP address or hostname of the RGW.
the region is what we call the "zonegroup", which is "default" if you don't set it to something.

Reply:
Sorry, I forgot to update the flag.